packages/quantic/force-app/main/default/lwc/quanticCarousel/quanticCarousel.html

```diff
@@ -0,0 +1,48 @@
+<template>
+  <template if:true={errorMessage}>
+    <c-quantic-component-error
+      component-name={template.host.localName}
+      message={errorMessage}
+    >
+    </c-quantic-component-error>
+  </template>
+  <template if:false={errorMessage}>
+    <section aria-roledescription="carousel" aria-label="carousel">
```

Comment:
Just noticed this note on the w3c example:
> NOTE: The aria-roledescription value is defined using the language of the page and should be localized by the author.

Probably that both of these labels should be localized.

The `roledescription` should probably stay something simple like "Carousel".

The label should probably be something more specific like "Recommendations".

Reply:
Should probably also add `role="region"` here (based on the same w3c docs)

Reply:
will do👌